transcom/mymove
# B 20012 INT GBLOC Endpoint Followup

## [B-20012](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-20012)

## Summary

Updates the GBLOC Switcher to use a list of distinct GBLOCs from the transportation_offices table.

## Verification Steps for the Author

These are to be checked by the author.


## Verification Steps for Reviewers

These are to be checked by a reviewer.


### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/getting-started/application-setup/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/getting-started/development/testing)

### How to test

1. Access the office app as an HQ user.
2. Click around, switch GBLOCs, refresh the page, etc. making sure it functions the same as before.
3. Open up your DB viewer and get a list of GBLOCs with the following query: `SELECT DISTINCT gbloc FROM public.transportation_offices ORDER BY gbloc ASC`
4. Verify the list returned matches the list available in the dropdown.

### Frontend


### Backend


### Database

#### Any new migrations/schema changes:


## Screenshots
